api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -463,6 +465,22 @@ public void validateTermsOfService(@Nonnull DbUser dbUser) {
     validateTermsOfService(tosVersion);
   }
 
+  @Override
+  public boolean validateTerraTermsOfService(@Nonnull DbUser dbUser) {
```

Comment:
Since this is a pass-through to `getUserTermsOfServiceStatus()`, that would be a good name for this method too.